Backend/DataAccessLayer/DataAccessLayer.csproj

```diff
@@ -49,15 +50,12 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Database\dbContext.cs" />
-    <Compile Include="Migrations\201812040636053_create-table-user.cs" />
-    <Compile Include="Migrations\201812040636053_create-table-user.Designer.cs">
-      <DependentUpon>201812040636053_create-table-user.cs</DependentUpon>
-    </Compile>
-    <Compile Include="Migrations\201812040642468_create-table-session.cs" />
-    <Compile Include="Migrations\201812040642468_create-table-session.Designer.cs">
-      <DependentUpon>201812040642468_create-table-session.cs</DependentUpon>
+    <Compile Include="Migrations\201903062125096_initial-create.cs" />
+    <Compile Include="Migrations\201903062125096_initial-create.Designer.cs">
+      <DependentUpon>201903062125096_initial-create.cs</DependentUpon>
```

Comment:
This migration should not be added, and the above migrations should not be removed